docs/desktop/gui/themes.md

```diff
@@ -420,7 +420,7 @@ result is identical to the real alias
 
 When extending themes the so-named `base` flag can be enabled to include the
 result of this selector of the derived theme into the local selector. This is
-quite comparable to the `this.base(arguments, ...)` call in member functions of
+quite comparable to the `super(...)` call in member functions of
```

Comment:
Line 421 is talking about `base` so I'm not sure this makes sense simply replacing `this.base` with `super` in this case...

Reply:
@johnspackman : What do you think?

Reply:
Line 421 is talking about the `base` keyword in theme definitions, which is still called `base`